views/default/elgg/i18n.js

```diff
@@ -16,7 +16,7 @@ define(['jquery', 'elgg', 'sprintf'], function ($, elgg, sprintf) {
 		 * @param {String} lang The language key (defaults to the current language)
 		 */
 		initLanguage: function(lang) {
-			language = lang || elgg.config.current_language;
+			var language = lang || elgg.config.current_language;
```

Comment:
can't we default lang in the function params? or is language something else than lang

Reply:
should be the same, changed somewhat